code/style.css

```diff
@@ -124,16 +154,61 @@ label {
   font-size: 16px;
   font-family: 'Montserrat';
   font-weight: 500;
-  color: #0026ff;
+  color: #ab82b8;
   margin-right: 20px;
 }
 
 button {
-  background-color: #0026ff;
+ 
+  background: rgba(255, 255, 255, 0.41);
+  border-radius: 16px;
+  box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+  backdrop-filter: blur(12.9px);
+  -webkit-backdrop-filter: blur(12.9px);
+  border: 1px solid rgba(255, 255, 255, 0.32);
+
+  color: white;
+  border: none;
+  border-radius: 10px;
+  padding: 16px 18px;
+  margin-right: 4px;
+  font-size: 16px;
+  line-height: 26px;
+  font-family: 'Montserrat';
+  font-weight: 500;
+  cursor: pointer;
+  transition: all 0.3s ease;
+}
+
+button:hover {
+  opacity: 0.9;
+  transition: all 0.2s ease;
+}
+
+#chosenFortune {
+  color: #fff3d9e1;
+  font-family: "Metamorphous";
+  font-size: 30px;
+  text-align: center;
+  line-height: 52px;
+  padding-top: 100px;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.dropDown {
+
+  background: rgba(255, 255, 255, 0.41);
+  border-radius: 16px;
+  box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+  backdrop-filter: blur(12.9px);
+  -webkit-backdrop-filter: blur(12.9px);
+  border: 1px solid rgba(255, 255, 255, 0.32);
+
   color: white;
```

Comment:
For me, the dropDown worked as intended, but I couldn't see the number on the options. Might changing color solve that?